packages/origin-ui-core/src/components/bundles/BundleTable.tsx

```diff
@@ -0,0 +1,161 @@
+import React, { useEffect, useState } from 'react';
+import {
+    TableMaterial,
+    IPaginatedLoaderHooksFetchDataParameters,
+    IPaginatedLoaderFetchDataReturnValues,
+    usePaginatedLoaderFiltered,
+    usePaginatedLoaderSorting
+} from '../Table';
+import { useTranslation, formatCurrencyComplete } from '../../utils';
+import { useSelector } from 'react-redux';
+import { getConfiguration, getProducingDevices } from '../..';
+import { Bundle } from '../../utils/exchange';
+import { getUserOffchain } from '../../features/users/selectors';
+import BN from 'bn.js';
+import { Visibility } from '@material-ui/icons';
+import { BundleDetails } from './BundleDetails';
+import { getCurrencies } from '../../features';
+import { getBundles } from '../../features/bundles/selectors';
+
+const BUNDLES_PER_PAGE = 25;
```

Comment:
We already have DEFAULT_PAGE_SIZE defined in https://github.com/energywebfoundation/origin/blob/a72cc1a51430c78a61aedc9aa23ea415aeae63a7/packages/origin-ui-core/src/components/Table/PaginatedLoaderHooks.tsx#L11

I don't think there's a need to set this to 25 if it already defaults to it.

Reply:
DEFAULT_PAGE_SIZE in Table component defined for all Table in the application and can be changed in the future while DEFAULT_PAGE_SIZE in `BundlesTable` should stay 25

Reply:
Is there a reason that it should always be 25 and not changeable?

Reply:
Its required in task description - https://energyweb.atlassian.net/browse/PTTDEV-693